src/utils/database.ts

```
@@ -1,16 +1,19 @@
   throw new Error("Please define the MONGODB_URI environment variable inside .env.local")
-}
+})()
 
 /**
  * Global is used here to maintain a cached connection across hot reloads
  * in development. This prevents connections growing exponentially
  * during API Route usage.
  */
+declare global {
+  // eslint-disable-next-line no-var
+  var mongoose: any
```

Biome (2.1.2)

[error] Shouldn't redeclare 'mongoose'. Consider to delete it or rename it.

'mongoose' is defined here:


(lint/suspicious/noRedeclare)